packages/webdriverio/src/commands/browser/waitUntil.js

```diff
@@ -29,6 +29,7 @@
  * @param {Number=}  timeout    timeout in ms (default: 500)
  * @param {String=}  timeoutMsg error message to throw when waitUntil times out
  * @param {Number=}  interval   interval between condition checks (default: 500)
+ * @return {Boolean} true if condition is fulfilled
```

Comment:
This is false since `timer.catch()` (Promise catch handler) will return undefined in all situations